### ISSUE TITLE
Support for MacOS environment

### ISSUE THREAD

**Issue by pprettytony** [appium/appium-flutter-driver#367]
**Stack:**
```
Maven 3.8.6
Java 11
MacOS 13.0
Appium 2.0.0-beta.46
```
**Description:**

I have a project, where I need to handle automation for Windows/Android/MacOS. Basically, I don't face any issues with this project, except MacOS. The main problem is that I can't inspect the page source of the desktop app on MacOS. I tried to setup `appium-flutter-driver` and run tests in Flutter context by setting `automationName:Flutter` capability, but it seems that there is no support for this context on MacOS and I facing this message:

`Lower-level error: Driver 'flutter' supports automationName 'Flutter', but Appium could not find support for platformName 'mac'. Supported platformNames are: ["iOS","Android"])`

The next step was to try different `flutter-finders`. I downloaded different versions for my project, tried to use them but there was no result, I can locate something, but I'm not even sure that I found a real element, always get something like this:

`[pro.truongsinh.appium_flutter.finder.FlutterElement@62942fe9 -> unknown locator]`

And no interactions with this element were successful.

Do we have any workaround for this environment? Maybe this question is more related to `appium-mac2-driver` but I suggest that it's something from Flutter-side support because all MacOS desktop automation actions seems to work properly (at least, I can start the app :) ). I know that I might miss something, maybe something should work and I just didn't find the correct stack of libs for resolving this issue.

**To summarise, is there any possibility to reach the elements from the Flutter-based application on MacOS automation environment?**

**Comment by thuonglai** [comment]
I think you need install appium-mac-driver to use platformName 'mac'

**Comment by KazuCocoa** [comment]
Plus, https://github.com/appium-userland/appium-flutter-driver/tree/main/driver/lib/sessions needs implementation to find the observatory URL to attach to the Dart VM

**Comment by KazuCocoa** [comment]
Duplicate of https://github.com/appium/appium-flutter-driver/issues/326
